<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCVDemoClass</class>
 <widget class="QMainWindow" name="OpenCVDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCVDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item row="0" column="1">
       <widget class="QGraphicsView" name="graphicsView_2"/>
      </item>
      <item row="1" column="0">
       <widget class="QGraphicsView" name="graphicsView_3"/>
      </item>
      <item row="1" column="1">
       <widget class="QGraphicsView" name="graphicsView_4"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_Gray">
        <property name="text">
         <string>灰度图</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_ImgFormat_Canny"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_Canny">
        <property name="text">
         <string>生成边界</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_Downsampling"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>降采样</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_ImgFormat_Gray"/>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_ImgName"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_SelectImg">
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_ImgFormat"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OpenCVDemo.qrc"/>
 </resources>
 <connections/>
</ui>
